File: backend/src/main/java/com/sistemaOficina/backend/entidade/ItensPeca.java
package com.sistemaOficina.backend.entidade;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ItensPeca {
    private int id;
    private double precoTotal;
    private int quantidade;
    private OrdemServico numeroOs;  
    private Pecas idPeca;
}

File: frontend/src/app/models/ItensPeca.ts
import { OrdemServico } from "./Os";
import { Peca } from "./Peca";

export interface ItensPeca {
    id: number;
    precoTotal: number;
    quantidade: number;
    numeroOs: OrdemServico; // Relacionamento com OrdemServico
    idPeca: Peca; // Relacionamento com Pecas
  }
  